shell/gpu/gpu_canvas.h

```diff
@@ -5,11 +5,29 @@
 #ifndef SHELL_GPU_GPU_CANVAS_H_
 #define SHELL_GPU_GPU_CANVAS_H_
 
+#include "flutter/shell/common/platform_view.h"
 #include "lib/ftl/macros.h"
+#include "lib/ftl/compiler_specific.h"
+#include "third_party/skia/include/core/SkCanvas.h"
 
 namespace shell {
 
-//
+class GPUCanvas {
+ public:
+  static std::unique_ptr<GPUCanvas> SelectPlatformCanvas(
```

Comment:
I imagined there would be a variety of options on the platform and this method would "select" a suitable one. Create works too. Patching.